rdf_differ/defaults.py

```diff
@@ -0,0 +1,12 @@
+"""
+defaults.py
```

Comment:
Do we really need an entire module to hold some default values that are used only once? 🤔